src/components/Image/__tests__/Image.test.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+import {render, screen} from '@testing-library/react';
+
+import {
+    testCustomClassName,
+    testCustomStyle,
+    testOnClick,
+} from '../../../../test-utils/shared/common';
+import {testSourceProps} from '../../../../test-utils/shared/image';
+import Image, {ImageProps} from '../Image';
+import i18n from '../i18n';
+
+const qaId = 'image-component';
+
+const imageSrc =
+    'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-gray.png';
+
+describe('Image', () => {
+    test('Render Image by default', async () => {
+        render(<Image src={imageSrc} qa={qaId} />);
+
+        const component = screen.getByTestId(qaId);
+        expect(component).toBeInTheDocument();
+        expect(component).toBeVisible();
+    });
+
+    test('add image as src prop', () => {
+        testSourceProps<ImageProps>({
+            component: Image,
+            props: {src: imageSrc, qa: qaId},
+        });
+    });
+
+    test('add image as desktop prop', () => {
+        testSourceProps<ImageProps>({
+            component: Image,
+            props: {desktop: imageSrc, qa: qaId},
+        });
+    });
+
+    test('add image as teblet prop', () => {
+        testSourceProps<ImageProps>({
+            component: Image,
+            props: {src: imageSrc, tablet: imageSrc, qa: qaId},
```

Comment:
why are we using here `src` if we have `tablet`?

Reply:
Because we do not render component if `src` or `desktop` are not provided
https://github.com/gravity-ui/page-constructor/blob/main/src/components/Image/Image.tsx#L42

Reply:
Tablet supposed to use with desktop, not with src. But maybe it is a props mistake not yours